source/retryable-reads/tests/aggregate.json

```diff
@@ -12,7 +12,8 @@
       "topology": [
         "sharded",
         "load-balanced"
-      ]
+      ],
+      "serverless": "forbid"
```

Comment:
These tests aren't failing for me.  Do they need to be disabled?

Reply:
You're right, these tests set a fail point, so they will never trigger the underlying error. I've removed the requirement for this test.